util/testutil/integration/run.go

```diff
@@ -216,6 +217,13 @@ func copyImagesLocal(t *testing.T, host string, images map[string]string) error
 				return err
 			}
 		}
+
+		// already exists check
```

Comment:
fyi, this shouldn't affect rate-limiting as HEAD requests are free, but no reason to recheck if tag has been updated in here